presto-main/src/test/java/com/facebook/presto/server/TestQueryResource.java

```diff
@@ -103,12 +98,64 @@ public void testGetQueryInfos()
         assertStateCounts(infos, 0, 0, 0);
     }
 
+    @Test(timeOut = 60_000)
+    public void testExpiredQueryInfo()
+            throws Exception
+    {
+        runToQueued("SELECT 1");
+
+        Thread.sleep(TimeUnit.SECONDS.toMillis(40));
```

Comment:
Can we have lover timeouts? e.g.: `10s` / `20s`?

Reply:
Sure

Reply:
nit: static import `Thread.sleep` and `TimeUnit.SECONDS`